homeassistant/components/goodwe/number.py

```diff
@@ -25,6 +25,7 @@ class GoodweNumberEntityDescriptionBase:
 
     getter: Callable[[Inverter], Awaitable[int]]
     setter: Callable[[Inverter, int], Awaitable[None]]
+    filter: Callable[[Inverter], bool]
```

Comment:
Are the changes in this file related to this pull request?

Reply:
No, not directly.
It was not formally filed as bug and reported only in user forum and I did not want to create another PR and wait few more months to get it in. 
It can be separated if you insist, but I considered it a small bugfix which could be piggybacked int this change.

Reply:
If you put it as a separate bugfix, we can get it in in point realeses. So please separate.

Reply:
I did not meant wait for major release, but wait for review (which takes months ;-)
I will separate it, hoping it will get reviewed sooner ...

Reply:
https://github.com/home-assistant/core/pull/77290